WordPress/src/main/java/org/wordpress/android/ui/people/PeopleManagementActivity.java

```diff
@@ -177,6 +181,11 @@ public void onPersonSelected(Person person) {
 
     @Override
     public void onRoleChanged(long personID, int localTableBlogId, String newRole) {
+        if(!NetworkUtils.isNetworkAvailable(this)) {
+            ToastUtils.showToast(this, R.string.error_update_role, ToastUtils.Duration.LONG);
```

Comment:
Since the error is due to no network, it'd be nicer to tell the user that's what the problem is. How about `R.string.no_network_message` instead?